cli/cmd/create_clusters.go

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
```

Comment:
check license - it should look like this -- https://github.com/GoogleCloudPlatform/kubernetes-engine-samples/blob/master/hello-app/main.go (with current year)